aws/eks-anywhere-build-tooling
# jgw/release06 sync

- Remove redundant yum package installation (#145)
- Fix filepath for mounted SSH key secret (#147)
- Image version to test (#146)
- Install QEMU during build setup (#148)
- Standardize patching, artifacts and binary targets (#151)
- [PR BOT] Update ATTRIBUTION.txt files (#152)
- cilium: convert to common.mk (#153)
- capv: switch to common.mk (#154)
- diag-image: switch to common.mk (#156)
- vsphere-csi-driver: switch to common.mk (#157)
- Copy top-level manifests folder when pushing to S3 (#159)
- cloud-provider-vsphere: convert to common.mk and introduce release br… (#158)
- Make buildspecs consistent, add artifacts upload target to filter (#160)
- Override image component for diagnostic collector (#161)
- Build S3 artifacts in release make target (#164)
- Add target to upload source controller artifacts (#165)
- Add upload artifacts script for source controller project (#166)
- bootlerocket-bootstrap: use common.mk (#162)
- Override image component for Bottlerocket bootstrap image (#167)
- image-builder: improve ova built stability (#168)
- source-controller: switch to common.mk (#169)
- Add version info to generated list, update tags in README (#170)
- Enable packer logs during OVA build (#176)
- Force-delete existing VMs of same name during OVA build (#177)
- Revert "image-builder: improve ova built stability (#168)" (#178)
- kind: switch to common.mk (#163)
- [PR BOT] Update base image tag file(s) (#175)
- Run local registry container in detached mode (#179)
- kind: use local registry for intermediate base images
- Fix tagging for kind node image dependencies (#180)
- kind: fix image taging and tarball directory (#183)
- attribution: no need to build binaries just need go mod download (#182)
- Fix replacement of embedded images in kind node image (#185)
- kind: use proper etcdadm arch binary (#186)
- fix attribution target for the typical ATTRIBUTION.txt targets (#184)
- eks-a-tools: switch to common.mk (#187)
- cli-tools: fix image component (#190)
- kind: improve base image crictl hash check (#189)
- kind: fix embedded arm images (#191)
- etcadm: fix pull arm64 tar (#193)
- we only need s3 artifacts for a few projects (#192)
- kind: arm64 support for etcd when run with etcdadm (#194)
- kind: etcd apply arm support to final image
- fetch-binaries: move to common.mk and update projects (#195)
- buildkit: enable inline caching in latest image (#196)
- Revert "Revert "image-builder: improve ova built stability (#168)" (#178)" (#188)
- move adding users/groups to container image to common (#197)
- kindnetd: extract manifest for use in capd cluster (#199)
- kindnetd: fix manifest targets and artifacts (#200)
- cli-tools: add sleep binary for reusing the container (#202)
- only export image cache when pushing images (#201)
- update checksums in attribution periodic (#204)
- small changes to common scripts to better match eks-distro (#207)
- [PR BOT] Update base image tag file(s) (#209)
- remove release-upload and use release in buildspecs (#203)
- create-pr: handle dry-run with multiple changes (#210)
- image-builder: switch to use common.mk and remove 1-18 and 1-19 (#205)
- fix image-builder and kind (#214)
- Run artifacts upload in dry-run mode outside of Codebuild (#215)
- image-builder: fix builds and clean up artifact uploads (#217)
- Remove unnecessary script invocation (#218)
- Add packer log to list of expected artifacts (#219)
- [PR BOT] Update base image tag file(s) (#220)
- Update capv to v1.0.1 (#216)
- Mark CAPV as producing S3 artifacts (#221)
- Revert "Update capv to v1.0.1 (#216)" (#224)
- [PR BOT] Update base image tag file(s) (#227)
- Break helm build and push into two scripts (#226)
- Add pvc to chart (#150)
- eks-anywhere-test: cleanup attribution handling for helm chart (#232)
- more sync changes from eks-distro (#208)
- fetch-binaries: use latest tag when deciding s3 folder instead of jus… (#240)
- Fix OVA S3 upload destinations (#242)
- Remove trailing parenthesis in script (#244)
- support image caching in kind builds (#234)
- source-controller: fix cgo builder image to be builder-base (#247)
- etcdadm: fix version by setting proper ldflags (#248)
